### ISSUE TITLE
Surprising result for #🏳️‍⚧️

### ISSUE THREAD

**Issue by hober** [mathiasbynens/hashtag-regex#3]
```
> const hashtagRegex = require('hashtag-regex');
undefined
> hashtagRegex().exec("#🏳️‍⚧️");
[ '#🏳️‍', index: 0, input: '#🏳️‍⚧️', groups: undefined ]
```

**Comment by hober** [comment]
#🏳️‍🌈 works fine:

```
> hashtagRegex().exec("#🏳️‍🌈");
[ '#🏳️‍🌈', index: 0, input: '#🏳️‍🌈', groups: undefined ]
```

**Comment by mathiasbynens** [comment]
Good catch! This is an old repository/project, and it looks like https://unicode.org/reports/tr31/#hashtag_identifiers has been updated since it was created. Updating the code to align with the new definition seems to fix this issue!

**Comment by mathiasbynens** [comment]
Fixed in https://github.com/mathiasbynens/hashtag-regex/commit/20d3f80e6bab409bb52fcdbae518dea5050b0086 and released in v2.2.0.
